apps/www/components/Auth/EmailForm.js

```diff
@@ -25,7 +25,6 @@ const styles = {
     flexGrow: 1,
   }),
   button: css({
-    width: 160,
```

Comment:
Fyi: tested and doesn't change layout if no custom buttonLabel is provided. Removing it allows us to add longer button labels to the Email from: eg. "Benachrichtigen"

Reply:
Are you sure there are no side-effects? If we just change it to minWidth we can ensure that the behaviour remains the same for short text but still allow for longer text.

Reply:
I am pretty sure. minWdith would also prevent shorter labels to look consistent

Reply:
Well until now the width was set to 160 fixed, so for labels that take up less space, the buttons would look different with this change.